### PR TITLE
Fix Install pip error in Linux (Debian/Ubuntu) 

### PR DESCRIPTION
**Pull request by aris-py**
## Description
- Fix to Error: Command '['/home/MyPC/Applio-3.2.5/.venv/bin/python3.10', '-m', 'ensurepip', '--upgrade', '--default-pip']' returned non-zero exit status 1.
./run-install.sh: 36: .: cannot open .venv/bin/activate: No such file

## How has this been tested?

- Tested locally in Ubuntu 24.04.1

## Types of changes
- ensurepip was changed to get-pip.py in case of error
- Now FFmpeg will be installed using sudo

### CODE REVIEW

File: run-install.sh [hunk 1]
```diff
@@ -11,42 +11,102 @@ prepare_install() {
...
-        python -m pip install "pip<24.1"
-        echo
-        echo "Installing Applio dependencies..."
-        python -m pip install -r requirements.txt
-        python -m pip install torch==2.3.1 torchvision torchaudio --upgrade --index-url https://download.pytorch.org/whl/cu121
-        finish
     fi
+
+    python -m pip uninstall torch torchvision torchaudio -y
+    python -m pip install torch==2.1.1 torchvision==0.16.1 torchaudio==2.1.1 --index-url https://download.pytorch.org/whl/cu121
```
[the remark this answers is not in the document]
Comment (aris-py) [reply run-install.sh:103]: oh
i will fix it

File: run-install.sh [hunk 2]
```diff
@@ -11,42 +11,102 @@ prepare_install() {
...
+    fi
+
+    echo "Installing Applio dependencies..."
+    python -m pip install -r requirements.txt
+    if [ $? -ne 0 ]; then
+        echo "Error installing Applio dependencies."
+        exit 1
+    fi
+
+    # try to install pyworld
```
[the remark this answers is not in the document]
Comment (aris-py) [reply run-install.sh:90]: gave me mistakes if I don't put it on, if you want I take it off

[the remark this answers is not in the document]
Comment (blaisewf) [reply run-install.sh:90]: quitalo, no se usa

[the remark this answers is not in the document]
Comment (aris-py) [reply run-install.sh]: arreglado

**Pull request merged**